frontend/src/main/web/user-profile-page/README.md

```diff
@@ -0,0 +1,15 @@
+# React Profile
+
+## Setup
+
+`npm install`
+
+## Options
+
+### Production Build
+
+`npm run build`
+
+### Development Server
```

Comment:
Okay, thanks for the clarification.  Could you please put that in the readme?

Is the server also serving up fake REST responses?